creator-node/src/snapbackSM/snapbackSM.js

```diff
@@ -92,13 +107,15 @@ class SnapbackSM {
       return
     }
 
+    // Setup the mapping of Content Node endpoint to service provider id. Used in reconfig
+    await this.updateEndpointToSpIdMap()
```

Comment:
why do this in the `init()` fn if it is refreshed every `processStateMachineOperation()` run? would be cleaner to just call it in each run and return the value instead of setting a class var

Reply:
this + healthyNodes do not need to be persisted / can just be passed in via params